pkg/blob/blob.go

```diff
@@ -0,0 +1,96 @@
+package blob
+
+import (
+	"bytes"
+	"errors"
+	fmt "fmt"
+	math "math"
+	"sort"
+
+	"github.com/celestiaorg/celestia-app/pkg/namespace"
+)
+
+// ProtoBlobTxTypeID is included in each encoded BlobTx to help prevent
+// decoding binaries that are not actually BlobTxs.
+const ProtoBlobTxTypeID = "BLOB"
```

Comment:
[unrelated to this PR, just a comment no change needed]

I know that we need something like this atm, but there _has_ to be better solution to this

Reply:
could maybe be solved with a flavor of https://github.com/celestiaorg/celestia-app/issues/1105